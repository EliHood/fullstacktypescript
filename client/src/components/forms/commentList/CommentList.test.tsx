// import React from "react";
// import CommentList from "./CommentList";
// import Grid from "@material-ui/core/Grid";
// import { createShallow } from "@material-ui/core/test-utils";
// import toJson from "enzyme-to-json";
// const props = {
//     comments: [
//         {
//             userId: 1,
//             id: 1,
//             comment_body: "delectus aut autem",
//             author: {
//                 username: "Bill",
//             },
//         },
//         {
//             userId: 2,
//             id: 2,
//             comment_body: "delectus aut autem",
//             author: {
//                 username: "Bill",
//             },
//         },
//         {
//             userId: 3,
//             id: 3,
//             comment_body: "delectus aut autem",
//             author: {
//                 username: "Bill",
//             },
//         },
//     ],
// };
// describe("Should render <CommentList/>", () => {
//     let wrapper;
//     let shallow;
//     beforeEach(() => {
//         shallow = createShallow();
//         wrapper = shallow(<CommentList {...props} />);
//     });

//     it("should render <CommentList/>", () => {
//         expect(wrapper.find(Grid)).toHaveLength(1);
//     });

//     it("should snap <CommentList/> component", () => {
//         // expect(toJson(wrapper)).toMatchSnapshot();
//     });
// });
