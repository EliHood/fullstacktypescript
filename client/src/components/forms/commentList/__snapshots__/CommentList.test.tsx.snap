// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render <CommentList/> should snap <CommentList/> component 1`] = `
<WithStyles(ForwardRef(Grid))>
  <div
    key="0"
  >
    <WithStyles(ForwardRef(List))
      style={
        Object {
          "paddingBottom": "20px",
        }
      }
    >
      <OurListItem>
        <WithStyles(ForwardRef(Typography))
          align="left"
          color="primary"
        >
          delectus aut autem
        </WithStyles(ForwardRef(Typography))>
      </OurListItem>
      <WithStyles(ForwardRef(Typography))
        align="left"
        style={
          Object {
            "padding": "0px 0px",
          }
        }
        variant="caption"
      >
        Bill
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        align="left"
        style={
          Object {
            "fontSize": "12px",
          }
        }
        variant="body1"
      >
        Today at 2:55 PM
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Divider))
        component="li"
        variant="fullWidth"
      />
    </WithStyles(ForwardRef(List))>
  </div>
  <div
    key="1"
  >
    <WithStyles(ForwardRef(List))
      style={
        Object {
          "paddingBottom": "20px",
        }
      }
    >
      <OurListItem>
        <WithStyles(ForwardRef(Typography))
          align="left"
          color="primary"
        >
          delectus aut autem
        </WithStyles(ForwardRef(Typography))>
      </OurListItem>
      <WithStyles(ForwardRef(Typography))
        align="left"
        style={
          Object {
            "padding": "0px 0px",
          }
        }
        variant="caption"
      >
        Bill
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        align="left"
        style={
          Object {
            "fontSize": "12px",
          }
        }
        variant="body1"
      >
        Today at 2:55 PM
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Divider))
        component="li"
        variant="fullWidth"
      />
    </WithStyles(ForwardRef(List))>
  </div>
  <div
    key="2"
  >
    <WithStyles(ForwardRef(List))
      style={
        Object {
          "paddingBottom": "20px",
        }
      }
    >
      <OurListItem>
        <WithStyles(ForwardRef(Typography))
          align="left"
          color="primary"
        >
          delectus aut autem
        </WithStyles(ForwardRef(Typography))>
      </OurListItem>
      <WithStyles(ForwardRef(Typography))
        align="left"
        style={
          Object {
            "padding": "0px 0px",
          }
        }
        variant="caption"
      >
        Bill
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Typography))
        align="left"
        style={
          Object {
            "fontSize": "12px",
          }
        }
        variant="body1"
      >
        Today at 2:55 PM
      </WithStyles(ForwardRef(Typography))>
      <WithStyles(ForwardRef(Divider))
        component="li"
        variant="fullWidth"
      />
    </WithStyles(ForwardRef(List))>
  </div>
</WithStyles(ForwardRef(Grid))>
`;
